aubreylib/resource.py

```diff
@@ -48,14 +48,16 @@ def get_desc_metadata(metadata_filename, metadata_type):
     """ Get the descriptive metadata for the object """
     # Open and read the metadata file into a StringIO filehandle
     metadata_filehandle = open_system_file(metadata_filename)
-    metadata_stringfile = StringIO.StringIO(metadata_filehandle.read())
+    metadata_stringfile = io.StringIO(metadata_filehandle.read())
     if metadata_type == 'UNTL':
         # Get the untl descriptive metadata dictionary
+        from pyuntl.untldoc import untlxml2pydict
```

Comment:
What was the purpose for moving all the pyuntl imports?

Reply:
To check when the test cases are failing. When the imports are in the beginning no test is running since the import urllib2 raises errors.

Reply:
This may be a case where we have to upgrade pyuntl before we can merge this in. I don't think the temporary benefit of this move outweighs the downside of not having the pyuntl imports show up with the rest of the module imports.

Reply:
I will move them back and keep with other imports once pyuntl is also upgraded. I just moved them within the code temporarily to check what is failing when trying to upgrade using 2to3 tool. I opened this pull request to just give an idea of how I'm proceeding on my work. I will clean everything up before merging it with the master.